examples/bri-3/src/bri/ccsm/contracts/Ccsm.sol

```diff
@@ -0,0 +1,25 @@
+//SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.17;
+
+contract Ccsm {
```

Comment:
name should probably be `ccsm-bpi-state-anchor`

Reply:
the contract needs access control:

```
import "@openzeppelin/contracts/access/AccessControl.sol";

contract MyContract is AccessControl {

  bytes32 public constant ADMIN_ROLE = keccak256("ADMIN_ROLE");

  constructor(address[] memory admins) {
    _setupRole(DEFAULT_ADMIN_ROLE, msg.sender); // Grant deployer the default admin role

    for (uint i = 0; i < admins.length; i++) {
      _grantRole(ADMIN_ROLE, admins[i]); // Grant admin role to each address
    }
  }

  // Your contract functions here

  modifier onlyAdmin() {
    require(hasRole(ADMIN_ROLE, msg.sender), "Only admin can call this function");
    _;
  }
```

and the change in `setAnchorHash` is 
```
function setAnchorHash(
    string calldata _workgroupId,
    string calldata _anchorHash
  ) external onlyAdmin
```

Reply:
Changes made.